### ISSUE TITLE
User test IA names

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4649]
Test our new proposed IA names using Treejack. E.g. https://bananacom.optimalworkshop.com/treejack/bananacom-demo-survey

### Todo (wip list)
- [x] Signup for account
- [x] Write script
- [x] Run test

**Comment by taisdesouzalessa** [comment]
@beccaklam  is the driver, @taisdesouzalessa is the supporter

**Comment by beccaklam** [comment]
@kristinashu @taisdesouzalessa Started a [brainstorm doc](https://docs.google.com/document/d/1iNvcfNO_l99XA2irKWDKfyOUNt2Xw_a8_yRcCfzrANQ/edit?usp=sharing) for Treejack tasks for us to use!

**Comment by beccaklam** [comment]
@kristinashu @taisdesouzalessa I've come up with three tasks for both novices/experts in [our brainstorm doc](https://docs.google.com/document/d/1iNvcfNO_l99XA2irKWDKfyOUNt2Xw_a8_yRcCfzrANQ/edit#bookmark=id.16lj2tuxgb6q). Can you take a look and let me know what you think before I start filling out the rest of the test? Thanks for all the helpful feedback already @taisdesouzalessa, it was super helpful!

**Comment by beccaklam** [comment]
@kristinashu @taisdesouzalessa So I've inputted most of the tasks, might need some help on Monday to go over what the "right answers" are. Also, treejack allows for pre and post study questions that allow for single answers. Any idea of what we may want to ask?

**Comment by taisdesouzalessa** [comment]
Awesome, Becca! I am looking forward to see this study.

For pre-study you can ask some things to warm them up and ease them into the study. It is also a good opportunity to get to know them better: are they tech-savvy, are they online a lot, or anything else it is relevant to know for your study. 

Post-study questions can include things related to how they feel after this exercise, how would they rate the experience, what was the easy part, what was the challenging part, if there is anything else they would like to add.

**Comment by beccaklam** [comment]
@taisdesouzalessa @kristinashu I've exported the "trees" and "tasks" for each test and uploaded the attachment to this comment. Would you be able to take a look and see if the "correct" answers I've selected for each task look good? Thank you!
[treejack-tests.zip](https://github.com/mozilla/foundation.mozilla.org/files/4678765/treejack-tests.zip)

**Comment by taisdesouzalessa** [comment]
@beccaklam here is my feedback, in blue "sticky notes" - https://mozilla.invisionapp.com/freehand/document/qFni9qIgB

**Comment by beccaklam** [comment]
@taisdesouzalessa Thanks for the feedback! I'll add 'Blog' to all the places you pointed out. As to keeping all the tests to the same depth -- I think that may limit us unnecessarily? The test is already quite simple and Treejack should record how a user got to an answer so I don't think timing really matters here.

**Comment by beccaklam** [comment]
Started a doc here for internal interview signups: https://docs.google.com/spreadsheets/d/1irTuJHqTWTVeJE2KaX1dI0cIKkNdNE_Z1w6W-y9MESk/edit?usp=sharing

**Comment by beccaklam** [comment]
Meeting with Abigail and Tais tomorrow to discuss legality of recruiting volunteers through "free" mediums.

**Comment by beccaklam** [comment]
Met with Abigail and Alicia Gray form MoCo today. They're okay with us proceeding if we use the paid recruitment panel Optimal Workshop provides. We just need to send Abigail and copy of the test we want to run beforehand.

**Comment by beccaklam** [comment]
Drafted new questions to add to task and set up the studies in Treejack [(see screen shots at bottom of this doc)](https://docs.google.com/document/d/1iNvcfNO_l99XA2irKWDKfyOUNt2Xw_a8_yRcCfzrANQ/edit?usp=sharing). Still waiting on cade to get credit card information.

**Comment by beccaklam** [comment]
Ran two tests with 10 participants, waiting for results to see if we made any major mistakes before ordering another two tests with 40 participants each. @kristinashu should I close this ticket or leave it open to move to next sprint?

**Comment by kristinashu** [comment]
Can you open a new ticket for next sprint? It can be about sharing the test results and decided if any changes are needed based on the results. Then this ticket is good to close!

**Comment by beccaklam** [comment]
New ticket here: #4867 Closing this one! 🥳
